### PR TITLE
Jekyllize the docs

### PR DESCRIPTION
**Pull request by fattarsi**
* Based on the changes from @jmeridth 's work on moving this to jekyll
* Easier to run locally
* Github pages supports it with no post-processing on our part
* Links to older versions of the docs were fixed and point to the github code viewer

### CODE REVIEW

**Comment by fattarsi** [comment]
Preview it at https://fattarsi.github.io/vaquero-docs/

**Review by jmeridth — APPROVED** [review]

File: _includes/page_head.html [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<head>
+  <meta charset="UTF-8">
+  <!--[if IE]><meta http-equiv="X-UA-Compatible" content="IE=edge"><![endif]-->
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Vaquero README</title>
```
Comment (jmeridth) [inline _includes/page_head.html]: Needs to be `{{ page.title }}`

File: _includes/head.html [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<head>
+    <meta charset="UTF-8">
+    <!--[if IE]><meta http-equiv="X-UA-Compatible" content="IE=edge"><![endif]-->
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="generator" content="Asciidoctor 1.5.2">
+    <title>Vaquero Docs</title>
```
Comment (jmeridth) [inline _includes/head.html]: Needs to be `{{ page.title }}` and we may want to set a default in configuration if one is not set on the markdown. https://jekyllrb.com/docs/configuration/#front-matter-defaults

**Review by jmeridth — CHANGES_REQUESTED** [review]

File: _includes/head.html (hunk 2 above)
Comment (fattarsi) [reply _includes/head.html]: Cool - good catch. Although, I couldn't figure out how to set the default in _config, I set it inline as I found here: https://github.com/jekyll/jekyll/issues/3046

File: _includes/page_head.html (hunk 1 above)
Comment (fattarsi) [reply _includes/page_head.html]: done

**Review by jmeridth — APPROVED** [review]

**Pull request merged**